pilot/pkg/bootstrap/server.go

```diff
@@ -274,6 +274,7 @@ func (s *Server) Start(stop <-chan struct{}) error {
 	}
 
 	// grpcServer is shared by Galley, CA, XDS - must Serve at the end, but before 'wait'
+	log.Infof("starting gRPC discovery service at %s", s.GRPCListener.Addr())
```

Comment:
move into the below go func?

Reply:
yeah. makes sense. Moved this and the below http log line in to go func